ecs/daemon/inputs/ecs/all-ecs-dataplane.conf

```diff
@@ -0,0 +1,12 @@
+[INPUT]
```

Comment:
I keep forgetting my own guidance here: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#tail-input-duplicates-logs-because-of-rotation

I need to double check and consider all of these. For each, there's a trade-off:
1. If we match rotated log files, then we will get duplicates
2. If we do not match all of the existing rotated log files at the time when Fluent Bit starts, then we won't get all of those old logs. Probably this is okay, but it'd be cool if there was a solution for it to get all the old logs. The ECS log collector script will grab all old logs IIRC: https://github.com/aws/amazon-ecs-logs-collector

Reply:
I think this is something to just note in the debugging/DIY part of the guide